src/pages/MapViewer/components/Toolbox/components/ToolboxRow/styles.scss

```diff
@@ -63,4 +64,19 @@
       }
     }
   }
+
+  &__copy-link {
+    position: absolute;
+    right: 10px;
+    bottom: 10px;
```

Comment:
Can't this not be better achieved using flex and justify-content? That way it stays within the normal rendering of an element and not escape from it.